pkg/tracker/tracker.go

```diff
@@ -51,7 +51,11 @@ func init() {
 	if err != nil {
 		panic(err)
 	}
-	Tracker = &tracker{gaPropertyID: os.Getenv(GA_TRACKING_ID), cid: id.String()}
+	Tracker = &tracker{
```

Comment:
Lets move this code out of init and define a function to create a tracker.

```
NewTracker() error
```

Reply:
done